lib/generators/fae/templates/graphql/graphql_type.rb

```diff
@@ -5,8 +5,10 @@ class Types::<%= class_name %>Type < Types::BaseObject
   field :id, ID, null: false
 <% if @graphql_attributes.present? -%>
 <% @graphql_attributes.each do |graphql_object| -%>
+<% if graphql_object[:attr] != :static_page -%>
```

Comment:
The above suggestion would also nullify this change, because without `BaseFaePageType` we'd want to keep the the auto-generation of static page GQL.